### PR TITLE
Enable mixed (inequality+equality) Design constraints for the LP

### PR DESCRIPTION
**Pull request by chengmiao0808**

### CODE REVIEW

File: dualip/src/main/scala/com/linkedin/dualip/solver/MooDistributedRegularizedObjective.scala [hunk 1]
```diff
@@ -58,13 +58,14 @@ abstract class MooDistributedRegularizedObjective(b: SparseVector[Double], gamma
...
     * Implements the main interface method
-    * @param lambda   The variable (vector) being optimized
-    * @param log      Key-value pairs used to store logging information for each iteration of the optimizer
-    * @param verbosity  Control the logging level
-    * @param designInequality True if Ax <= b, false if Ax = b
+    * @param lambda               The variable (vector) being optimized
+    * @param log                  Key-value pairs used to store logging information for each iteration of the optimizer
+    * @param verbosity            Control the logging level
+    * @param designInequality     True if Ax <= b, false if Ax = b or have mixed constraints
+    * @param mixedDesignPivotNum  The pivot number if wwe have mixed A_1x <= b1 and A_2x = b2, i.e. how many inequality constraints come first
```
Comment (basukinjal) [inline dualip/src/main/scala/com/linkedin/dualip/solver/MooDistributedRegularizedObjective.scala:65]: Typo in `wwe`

**Review by basukinjal — CHANGES_REQUESTED** [review]
Minor comments added. Both the `designInequality` and the `pivotNum` are being added to only AGD. Can we add a TODO list to add it to the other algorithms as well?

File: dualip/src/main/scala/com/linkedin/dualip/solver/MooDistributedRegularizedObjective.scala (hunk 1 above)
Comment (chengmiao0808) [reply dualip/src/main/scala/com/linkedin/dualip/solver/MooDistributedRegularizedObjective.scala:65]: oh, thanks for the catch.

**Comment by chengmiao0808** [comment]
> 

Have added 'TODO' comment.

**Review by aacharya-li — APPROVED** [review]

**Pull request merged**